doc/modules/mod_pubsub.md

```diff
@@ -109,7 +101,7 @@ Requires module `mod_caps` to be enabled.
 #### `<<"dag">>`
 
 Implementation of [XEP-0248 (PubSub Collection Nodes)](https://xmpp.org/extensions/xep-0248.html).
-Every node takes a place in a tree and is either a collection node (and have only sub-nodes) or a leaf node (contains only items).
+Every node takes a place in a collection and is either a collection node (and have only sub-nodes) or a leaf node (contains only items).
```

Comment:
Every node takes a place in a collection and becomes either a collection node (and have only sub-nodes) or a leaf node (contains only items).